README.md

```diff
@@ -10,13 +10,13 @@
 DAIDE parser using [parsimonious](https://github.com/erikrose/parsimonious). Parsimonious is a python package that uses "a simplified sort of EBNF notation" to define a grammar. The parser currently supports all 130 levels of DAIDE.
 
 - The original DAIDE specification is [here](daide-syntax.pdf)
-- The working markdown document that will included DAIDE enhancements is [here](daide-specification.md)
-- The machine-parsable grammar can be found in [this `.py` file](./src/daidepp/grammar.py)
+- The working Markdown document that will include DAIDE enhancements is [here](daide-specification.md)
+- The machine-parsable grammar can be found in [this `.py` file](./src/daidepp/grammar/grammar.py)
```

Comment:
I think I opened an issue on this...

https://github.com/SHADE-AI/daidepp/issues/93

Reply:
I don't have the access, so can you link the issue to this PR?